lib/spack/spack/database.py

```diff
@@ -44,7 +44,8 @@
 from spack.directory_layout import DirectoryLayoutError
 from spack.error import SpackError
 from spack.version import Version
-from spack.util.lock import Lock, WriteTransaction, ReadTransaction, LockError
+from spack.util.lock import \
+    Lock, LockError, LockTimeoutError, WriteTransaction, ReadTransaction
```

Comment:
change this to import `spack.util.lock as lk` and prefix the classes below -- we're trying to simplify imports in core, and not importing symbols directly makes it less likely you'll get circular import issues.

Reply:
Done